js/client-static/rpc/index.ts

```diff
@@ -0,0 +1,86 @@
+import fetch from "node-fetch";
```

Comment:
Should just use jayson and throw this out.

Reply:
Since jayson includes 'commander' as a dep, might as well replace the shitty CLI too